docs/user-docs/building.md

```diff
@@ -1,3 +1,6 @@
+
+[//]: # (TODO: Convert this page into links to project specific build instructions)
```

Comment:
Good to-do.

Reply:
As mentioned, this is not official. Its an abuse of markdowns reference style links. Ideally, its invisible on readthedocs but its hard to say until its actually pushed.